mlibrary/search
# [LIBSEARCH-878] Fix accessibility errors and warnings (Part 1)

# Overview
An [accessibility evaluation](https://docs.google.com/document/d/1LiRYT1Q_eaN4mJuOb70QIVHKemx4khs7cV19JCHynBY/edit?usp=sharing) was done on Search. This pull request resolves the majority of the violations and warnings that were found, along with resolving other finds via ARC Toolkit.

This pull request relates to [LIBSEARCH-878](https://mlit.atlassian.net/browse/LIBSEARCH-878).

## Anything else?
The issues relating to the modals and accordions are not included in this, as they require building new components.

## Testing
- Make sure the PR is consistent in these browsers:
- Run accessibility tests:
- Go through the site to see if anything is broken.
